scripts/KlipperScreen-install.sh

```diff
@@ -24,7 +24,9 @@ install_packages()
         virtualenv \
         matchbox-keyboard \
         wireless-tools \
-        libatlas-base-dev
+        libatlas-base-dev \
+        python3-gst-1.0 \
+        vlc
```

Comment:
I just pushed an update for the install script. Can you please modify this so that the packages are in PKGLIST? This is required for moonraker to be able to add the packages during the update process.

Reply:
done 👍